File: kitten-housing-care/src/main/java/org/jesperancinha/housing/data/OwnerDto.java
package org.jesperancinha.housing.data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Builder
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class OwnerDto {
    private String name;

    private String addres;
}

File: kitten-house-care/src/main/kotlin/org/jesperancinha/housing/data/OwnerDto.kt
package org.jesperancinha.housing.data

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Getter
import lombok.NoArgsConstructor

@Builder
@Getter
@NoArgsConstructor
@AllArgsConstructor
class OwnerDto {
    private val name: String? = null
    private val addres: String? = null
}